shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -955,7 +955,7 @@ object GlutenConfig {
       .internal()
       .doc("The maximum size of a single spill file created")
       .bytesConf(ByteUnit.BYTE)
-      .createWithDefaultString("20MB")
+      .createWithDefaultString("0MB")
```

Comment:
I would suggest to change to 1G, iceberg project compaction file default size is 1G

Reply:
Yes sounds convicing. 1G is good for most cases. Let's use it.

Hopefully we can get some time to create some benchmarkings to decide on a optimal value of such configs.